src/client/components/pages/Courses/MeetingModal.tsx

```diff
@@ -227,20 +248,66 @@ const MeetingModal: FunctionComponent<MeetingModalProps> = function ({
     if (isVisible) {
       setSaveError('');
       setMeetingModalFocus();
-      setAllMeetings(instanceMeetings);
+      if (!initialized) {
+        setAllMeetings(instanceMeetings);
+        setInitialized(true);
+      }
     } else {
       setCurrentEditMeeting(null);
       setShowRoomsData(null);
+      setInitialized(false);
```

Comment:
Rather than create this `initialized` state value, what if we just split the `setAllMeetings(instanceMeetings)` call off into a separate `useEffect` hook that only listens for changes in `isVisible` and `setAllMeetings`? It's not a huge difference, but it might make it a little easier to keep track of what's going to get updated when

Reply:
Sure I think that this works - I made the change in commit 274b016.